hbase-client/src/main/java/org/apache/hadoop/hbase/client/RawAsyncHBaseAdmin.java

```diff
@@ -4006,9 +4007,9 @@ private void getProcedureResult(long procId, CompletableFuture<Void> future, int
     }
   }
 
-  private CompletableFuture<List<OnlineLogRecord>> getSlowLogResponseFromServer(
+  private CompletableFuture<List<LogEntry>> getSlowLogResponseFromServer(
```

Comment:
Comment on admin interface has implications here. (_This needs an optional parameter to allow constrained clients to limit the size of the returned list of LogEntry._)